content/ember-released-4-7.md

```diff
@@ -0,0 +1,135 @@
+
+#### Features
+
+Ember.js 4.7 introduced 0 features.
+
+#### Deprecations
+
+Ember.js 4.7 introduced 0 deprecations.
+
+<!-- Block end -->
+
+For more details on changes in Ember.js 4.7, please review the [Ember.js 4.7.0 release page](https://github.com/emberjs/ember.js/releases/tag/v4.7.0).
+
+---
+
+## Ember Data
+
+Ember Data is the official data persistence library for Ember.js applications.
+
+### Changes in Ember Data 4.7
+
+Version 4.7 implements a "2.1" cache spec while deprecating the v1 cache spec. This spec differs somewhat from the [original 2.0 RFC](https://rfcs.emberjs.com/id/0461-ember-data-singleton-record-data), an updated RFC specifying the modifications will be published soon. Users looking to migrate to the 2.1 cache should expect that the finalized version of 2.1 enforces that the cache is always a singleton (vs merely allowed to be a singleton), and that data provided to the store and queries to the store from the cache should always contain identifiers in their stable form. This latter point is enforced in 4.7, the former is not yet enforced.
+
+#### Bug Fixes
+
+Ember Data 4.7 introduced 8 bug fixes and some tests and documentation improvements. Additional details are in the [release notes](https://github.com/emberjs/data/releases/tag/v4.7.1).
+
+- [#8089](https://github.com/emberjs/data/pull/8089) Unpin yarn for consumers
+- [#8108](https://github.com/emberjs/data/pull/8108) Fix earlier versions of node-14
+- [#8118](https://github.com/emberjs/data/pull/8118) Reopen deprecation should properly pass original arguments
+- [#8113](https://github.com/emberjs/data/pull/8113) Account for multi-options-per-identifer possibility
+- [#8109](https://github.com/emberjs/data/pull/8109) Add optional polyfill for crypto.randomUUID
+- [#8116](https://github.com/emberjs/data/pull/8116) Clear subscriptions once unsubscribed, don't unnecessarily churn on subscriptions
+- [#8128](https://github.com/emberjs/data/pull/8128) `backgroundReloads` should not affect `recordIsLoaded`
+- [#8159](https://github.com/emberjs/data/pull/8159) uuid polyfill logic (@runspired)
+
+#### Features
+
+Ember Data 4.7 introduced 2 features.
+
+- [#7955](https://github.com/emberjs/data/pull/7955) Explicit Polymorphic Relationship Support [emberjs/rfcs#793](https://rfcs.emberjs.com/id/0793-polymporphic-relations-without-inheritance)
+- [#8134](https://github.com/emberjs/data/pull/8134) The return values of hasMany relationships, peekAll, findAll and query are now proxies to native arrays and as such all native array APIs are now usable. These objects will act fully as if they are native arrays. Restrictions on immutability of the result of peekAll and query still apply ([RFC 846](https://rfcs.emberjs.com/id/0846-ember-data-deprecate-proxies)).
+
+#### Deprecations
+
+Ember Data 4.7 introduced 8 deprecations. Additional details are in the [release notes](https://github.com/emberjs/data/releases/tag/v4.7.1).
```

Comment:
while there are only 8 PRs implementing deprecations there are 13 total. They are:

- deprecate model.reopen [emberjs/rfcs#738](https://github.com/emberjs/rfcs/pull/738)
- deprecate early static [emberjs/rfcs#741](https://github.com/emberjs/rfcs/pull/741)
- deprecate importable helper functions [emberjs/rfcs#742](https://github.com/emberjs/rfcs/pull/742)
- deprecate non-promise promise-many-array APIs  [emberjs/rfcs#745](https://github.com/emberjs/rfcs/pull/745)
- deprecate v1-cache store APIs  [emberjs/rfcs#461](https://github.com/emberjs/rfcs/pull/461)
- deprecate v1-cache  [emberjs/rfcs#461](https://github.com/emberjs/rfcs/pull/461)
- deprecate using `A()` on RecordArray/ManyArray/PromiseManyArray  [emberjs/rfcs#846](https://github.com/emberjs/rfcs/pull/846)
- deprecate relationships without a specified type [emberjs/rfcs#739](https://github.com/emberjs/rfcs/pull/739)
- deprecate relationships without specified async behavior [emberjs/rfcs#739](https://github.com/emberjs/rfcs/pull/739)
- deprecate relationships without an explicit inverse  [emberjs/rfcs#739](https://github.com/emberjs/rfcs/pull/739)
- deprecate polymorphic relationships without explicit polymorphic flags [emberjs/rfcs#793](https://github.com/emberjs/rfcs/pull/793)
- deprecate promise-array and promise-object return values [emberjs/rfcs#846](https://github.com/emberjs/rfcs/pull/846)
- deprecate array-like APIs (non native array APIs) on array proxies [emberjs/rfcs#846](https://github.com/emberjs/rfcs/pull/846)

Reply:
Cool I'll change it to 13 and people could find the docs in the release notes correct?